src/DurableTask.SqlServer.AzureFunctions/SqlDurabilityProvider.cs

```diff
@@ -197,8 +197,24 @@ public override async Task RewindAsync(string instanceId, string reason)
             string storageConnectionString,
             out IScaleMonitor scaleMonitor)
         {
-            scaleMonitor = this.scaleMonitor ??= new SqlScaleMonitor(this.service, hubName);
+            SqlMetricsProvider sqlMetricsProvider = new SqlMetricsProvider(this.service);
+            scaleMonitor = this.scaleMonitor ??= new SqlScaleMonitor(this.service, hubName, sqlMetricsProvider);
             return true;
         }
+
+#if NETCOREAPP
```

Comment:
Why do we need the `#if NETCOREAPP`? Is there any problem if this code exists in both the `netstandard2.0` and `netcoreapp3.1` targets?

Reply:
I figured this out. This is necessary because our dependency chain doesn't give us access to `ITargetScaler` when targeting `netstandard2.0`.